api-metastore/src/main/java/org/zalando/nakadi/service/SubscriptionService.java

```diff
@@ -166,7 +168,7 @@ public Subscription updateSubscription(final String subscriptionId, final Subscr
 
         authorizationValidator.authorizeSubscriptionAdmin(old);
 
-        subscriptionValidationService.validateSubscriptionChange(old, newValue);
+        subscriptionValidationService.validateSubscriptionOnChange(old, newValue);
```

Comment:
```suggestion
        subscriptionValidationService.validateSubscriptionOnUpdate(old, newValue);
```

If it's called `CRUD`...